src/main/java/io/cdap/e2e/pages/actions/CdfGcsActions.java

```diff
@@ -73,4 +70,38 @@ public static void skipHeader() {
   public static void getSchema() {
     CdfGCSLocators.getSchemaButton.click();
   }
+
+  public static void delimiter() throws IOException {
+    CdfGCSLocators.delimiter.sendKeys(SeleniumHelper.readParameters("delimiter_@"));
+  }
+
+  public static void selectFormat(String formatType) throws IOException, InterruptedException {
+    CdfGCSLocators.format.click();
+    String format = SeleniumHelper.readParameters(formatType);
+    SeleniumHelper.waitAndClick(
+      SeleniumDriver.getDriver().findElement(By.xpath("//*[contains(text(),'" + format + "')]")));
```

Comment:
Please put all XPath in the locator class

Reply:
As per my knowledge, annotations do not support xpath having a variable value to be replaced. So these xpath are still here, but others I have moved it to locator class